README.md

```diff
@@ -270,13 +139,13 @@ I would like to thank...
 
 > Add at least 2 questions new developers would ask when they decide to use your project.
 
-- **[Question_1]**
+- **[Queries completed with erros?]**
 
-  - [Answer_1]
+  - [Kindly check if PostgreSQL is properly installed.]
 
-- **[Question_2]**
+- **[Cloned files cannot be opened ?]**
 
-  - [Answer_2]
+  - [File is .sql extension, you can open it using any text editor]
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] It would be better if we create the License file in order to fix the `404 error page not found` error
![Screenshot from 2023-07-10 09-30-44](https://github.com/Theodoraldo/vet-clinic-db/assets/46388113/ba536e67-fe12-4f7c-acf9-02165ca0ca24)